docs/content/configuration-language.mdx

```diff
@@ -818,7 +823,7 @@ The JSON syntax accepted by the <ProductName format={ProductNameFormat.ShortForm
 
 | Zanzibar           | <ProductName format={ProductNameFormat.ShortForm}/> JSON | <ProductName format={ProductNameFormat.ShortForm}/> DSL |
 | :----------------- | :------------------------------------------------------- | :------------------------------------------------------ |
-| `this`             | `this`                                                   | `self`                                                  |
+| `this`             | `this`                                                   | `[]`                                                  |
```

Comment:
Note that `[]` (as an empty set) is invalid. Maybe we can link to direct relationship type restrictions instead?

Reply:
Hmm...here, we directly refer to the syntax.  Maybe I will do something like `[<type1>,<type2>]` with link to the direct relationship type restrictions?